### ISSUE TITLE
Can't install on Ubuntu 20.04

### ISSUE THREAD

**Issue by juanhuttemann** [faressoft/terminalizer#112]
node version: v14.3.0
npm vserion: 6.14.5

Doesn't install:

```sh 
$ sudo npm install -g terminalizer

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
/usr/bin/terminalizer -> /usr/lib/node_modules/terminalizer/bin/app.js

> @faressoft/node-pty-prebuilt@0.9.0 install /usr/lib/node_modules/terminalizer/node_modules/@faressoft/node-pty-prebuilt
> prebuild-install || node scripts/install.js

prebuild-install WARN install EACCES: permission denied, access '/root/.npm'
gyp WARN EACCES current user ("nobody") does not have permission to access the dev dir "/root/.cache/node-gyp/14.3.0"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/terminalizer/node_modules/@faressoft/node-pty-prebuilt/.node-gyp"
gyp WARN install got an error, rolling back install
gyp WARN install got an error, rolling back install
gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/terminalizer/node_modules/@faressoft/node-pty-prebuilt/.node-gyp'
gyp ERR! System Linux 5.4.0-33-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/lib/node_modules/terminalizer/node_modules/@faressoft/node-pty-prebuilt
gyp ERR! node -v v14.3.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @faressoft/node-pty-prebuilt@0.9.0 install: `prebuild-install || node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @faressoft/node-pty-prebuilt@0.9.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-06-04T17_44_30_527Z-debug.log

```

Install:
```sh
sudo npm install -g terminalizer  --unsafe-perm
```

**Comment by vineeths96** [comment]
I too faced a very similar issue. Any help would be appreciated.

```
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
/usr/local/bin/terminalizer -> /usr/local/lib/node_modules/terminalizer/bin/app.js
npm WARN lifecycle The node binary used for scripts is /snap/bin/node but npm is using /snap/node/2799/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> @faressoft/node-pty-prebuilt@0.9.0 install /usr/local/lib/node_modules/terminalizer/node_modules/@faressoft/node-pty-prebuilt
> prebuild-install || node scripts/install.js

2020/06/07 19:05:06.795812 cmd_run.go:918: WARNING: cannot create user data directory: cannot create "/nonexistent/snap/node/2799": mkdir /nonexistent: permission denied
cannot create user data directory: /nonexistent/snap/node/2799: Permission denied
2020/06/07 19:05:06.818819 cmd_run.go:918: WARNING: cannot create user data directory: cannot create "/nonexistent/snap/node/2799": mkdir /nonexistent: permission denied
cannot create user data directory: /nonexistent/snap/node/2799: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @faressoft/node-pty-prebuilt@0.9.0 install: `prebuild-install || node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @faressoft/node-pty-prebuilt@0.9.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-06-07T13_35_07_409Z-debug.log
```

**Comment by seru1us** [comment]
I had a similar issue, don't use sudo with NPM for installations. I use Kubuntu 20.04 and had to change the permissions on a few files as well.

**Comment by ChanceNCounter** [comment]
I've just commented to add that asking users to install a program through npm, let alone *npm freaking global*, is the height of absurdity. I'm kinda horrified that so many users have gone for it.

I know that sounds harsh, but, frankly, I can't think of a polite way to put it. Never `sudo npm`, never `npm install -g`, never `sudo pip`.

Or, ideally, don't use your language's package/module manager to distribute user-facing software.

**Comment by weiss-d** [comment]
Error of another variation, but the same type on Pop!_OS 20.04 🙁
```bash
 $ npm install -g terminalizer
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN checkPermissions Missing write access to /usr/local/lib
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib'
npm ERR!  { [Error: EACCES: permission denied, access '/usr/local/lib']
npm ERR!   stack:
npm ERR!    'Error: EACCES: permission denied, access \'/usr/local/lib\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib' }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/weissd/.npm/_logs/2020-09-24T12_10_28_481Z-debug.log
```
`2020-09-24T12_10_28_481Z-debug.log` file contains the same message.

**Comment by juanhuttemann** [comment]
> I've just commented to add that asking users to install a program through npm, let alone _npm freaking global_, is the height of absurdity. I'm kinda horrified that so many users have gone for it.
> 
> I know that sounds harsh, but, frankly, I can't think of a polite way to put it. Never `sudo npm`, never `npm install -g`, never `sudo pip`.
> 
> Or, ideally, don't use your language's package/module manager to distribute user-facing software.

Missing the point here

**Comment by ChanceNCounter** [comment]
@juanhuttemann My point was that the correct fix to this issue is to package the program.

**Comment by dimisjim** [comment]
This did the trick for me:

assuming a clean npm installation
```
sudo mkdir -p /usr/local/lib/node_modules && sudo chown -R jim:jim /usr/local/lib/node_modules
npm install -g terminalizer
```

Then you should be able to run: `terminalizer --help`

But indeed as @ChanceNCounter said last year, it is better for the repo maintainer to distribute the binary straight away

**Comment by ChanceNCounter** [comment]
This seems to be abandonware.

**Comment by faressoft** [comment]
Fixed in #226
